toolbar.less

```diff
@@ -34,6 +22,7 @@
     }
 
     .htk-toolbar-panel {
+        @list-item-background-hover: #ffc;
```

Comment:
@ajnassar we should call these variables

`@color-description: color-code;`

So in this case, `@color-list-item-background-hover: #ffc;`

Reply:
I took care of it in a follow up https://github.com/hacktoolkit/htk-css-hacktoolkit/commit/54360f2be9314e75d6f3b5741dec6b4490a9ac51